app/controllers/api.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+module Api; end
```

Comment:
Fixes "Unable to autoload constant" Rails error due to old Rails version - constanthttps://stackoverflow.com/questions/27830874/actionviewmissingtemplate-error-when-using-modules